sdk/resourcemanager/azure-resourcemanager-keyvault/src/main/java/com/azure/resourcemanager/keyvault/models/Key.java

```diff
@@ -29,34 +29,68 @@
 /** An immutable client-side representation of an Azure Key Vault key. */
 @Fluent
 public interface Key extends Indexable, HasInnerModel<KeyProperties>, HasId, HasName, Updatable<Key.Update> {
-    /** @return the Json web key. */
+    /**
+     * Gets the Json web key.
+     *
+     * @return the Json web key.
+     */
     JsonWebKey getJsonWebKey();
 
-    /** @return the Json web key. */
+    /**
+     * Gets the Json web key.
+     *
+     * @return the Json web key.
+     */
     Mono<JsonWebKey> getJsonWebKeyAsync();
 
-    /** @return the key management attributes. */
+    /**
+     * Gets the key management attributes.
+     *
+     * @return the key management attributes.
+     */
     KeyProperties attributes();
 
-    /** @return application specific metadata in the form of key-value pairs. */
+    /**
+     * Gets application specific metadata in the form of key-value pairs.
+     *
+     * @return application specific metadata in the form of key-value pairs.
+     */
     Map<String, String> tags();
 
     /**
+     * Checks whether the key's lifetime is managed by key vault.
+     *
      * @return true if the key's lifetime is managed by key vault. If this is a key backing a certificate, then managed
      *     will be true.
      */
     boolean managed();
 
-    /** @return a list of individual key versions with the same key name */
+    /**
+     * GEts a list of individual key versions with the same key name.
```

Comment:
typo

Reply:
Fixed by the [commit#27fca5d](https://github.com/Azure/azure-sdk-for-java/pull/43603/commits/27fca5d91c9dd839287fbc7190aff26b62c180b6)